torch/csrc/jit/runtime/argument_spec.h

```diff
@@ -58,18 +59,18 @@ struct ArgumentInfo {
   unsigned requires_grad_ : 1;
   unsigned : 5;
   unsigned dim_ : 8;
-  unsigned device_ : 8;
```

Comment:
Huh, what does that mean?

Reply:
I moved `device_` to the back for alignment reasons.

Reply:
From playing with it a bit, these padding don't make any difference since we're using bitfield which does optimal padding already.
Can you please juste remove them all?